### ISSUE TITLE
Gang Equipment | Annoying Wiggling Interface

### ISSUE THREAD

**Issue by GoulartNogueira** [bitburner-official/bitburner-src#1409]
### TLDR
At Gang > Equipment, the interface wiggles based on money available.
To fix that, equipment should be shown as disabled (not hidden) when the money available is not enough.

### Problem
Sometimes I'm trying to manage my Gang's equipment.
But as my money oscillates, so does the interface. Making it very hard to use the UI.

With a good intention, the Gang > Equipment interface will only show the equipment you can afford.
But that causes the whole interface to shift up and down as my money triggers the next equipment available.
Screen shift (without user's direct input) is very annoying.
And it's a bad UI pattern: This even makes websites rank lower at Google.

This is specially critical at mid-end-game, when my gang is at "turbo" mode, generating billions per second. While my stocks algorithm is consuming those billions. So it's basically impossible to buy an augmentation.

### Solution

I'd like to have every equipment available at my screen, no matter how much money I have.
(of course, only those I haven't bought yet and inside the category selected)

In short:
All equipment I cannot buy should be shown as a disabled button, instead of hidden.
Never hide a button based on external dynamic volatile factors (such as money available).

**Comment by GoulartNogueira** [comment]
Notice the interface shifts so much it goes to another member!

Member928 > Member547 > Member928

Sample video:

[Screencast from 18-06-2024 13:31:19.webm](https://github.com/bitburner-official/bitburner-src/assets/7660643/2249ad3e-fa62-4010-9b1c-86029d34f6f6)

**Comment by YarikTH** [comment]
For me the same effect caused inability to select the desired equipment type. At the time I clicked on the desired combo box entry, UI changes and I select some other tab unintentionally.

**Comment by YarikTH** [comment]
BTW there is another graphical bug in gang UI (it can be more suitable for the separate issue though): when I have bought all the augmentation, UI showed me both "All items are bought" and "N billions till next augmentation". I have already finished BN2, so I can't provide the screenshot yet

**Comment by Kelenius** [comment]
This is resolved by #1151.
